databases/metadata/src/main/conversions/c192_2014090201.clj

```diff
@@ -58,16 +58,16 @@
 
 (def convert
   "Performs the conversion for database version 1.9.2:20140902.01"
-  ^{:cfg-file "donkey.properties"}
-  (fn [admin-db-spec donkey-cfg]
+  ^{:cfg-file "terrain.properties"}
+  (fn [admin-db-spec terrain-cfg]
     (println "Performing the conversion for" version)
-    (let [fs-cfg (fs-init/init (get donkey-cfg "donkey.irods.host")
-                               (get donkey-cfg "donkey.irods.port")
-                               (get donkey-cfg "donkey.irods.user")
-                               (get donkey-cfg "donkey.irods.pass")
-                               (get donkey-cfg "donkey.irods.home")
-                               (get donkey-cfg "donkey.irods.zone")
-                               (get donkey-cfg "donkey.irods.resc"))]
+    (let [fs-cfg (fs-init/init (get terrain-cfg "terrain.irods.host")
+                               (get terrain-cfg "terrain.irods.port")
+                               (get terrain-cfg "terrain.irods.user")
+                               (get terrain-cfg "terrain.irods.pass")
+                               (get terrain-cfg "terrain.irods.home")
+                               (get terrain-cfg "terrain.irods.zone")
+                               (get terrain-cfg "terrain.irods.resc"))]
```

Comment:
I'm not sure about updating a conversion for a previous version.
I think it makes sense to leave these as `donkey` configs in this 1.9.2 conversion.